### ISSUE TITLE
Possible issue with old bilibili embeds

### ISSUE THREAD

**Issue by Move-Forward39** [VocaDB/VocaDB-App#76]
Steps to replicate it:
Songs -> under Filter have Original Song with Luo Tianyi and Yuezheng Ling -> try one of the songs here
You might not need these filter settings, but it might make it easier to have something like this assuming the issue occurs with other songs as well.
What happens is it'll go back to the "logo screen" instead of displaying the video when selecting one of these songs.
I think the issue may be related to bilibili having updated their embed code. (There's a discussion here about it: https://vocadb.net/discussion/topics/182) These songs where the issue occurs seem to be songs with bilibili uploads for some time ago. The recent ones under Luo Tianyi work just fine, so I assume it's just older ones.
Occurs with Android. Unknown if iOS experiences the same thing or not.

**Comment by up2code** [comment]
This issue cause a song with bilibili pv not contain _extendedMetadata_ field that I had to used for display embed bilibili player and that make app crashed. I should check null on this field.

Actually this should be fixed on latest version (2.0.6) but with my mistake I had published the build version that not include this fix (fe836c9c5e1e0303773bc44375ccdd4a42bf7f0e) Also this issue found on iOS too.

I'll published the correct version soon for fix this issue (might along with some other minor bug fixed too) maybe in this weekend
